### PR TITLE
Install webpack

### PR DESCRIPTION
**Pull request by brytebee**
## Installations
In this part of the project, I install webpack and linters check both locally and remotelly.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+].sort((a, b) => a.index - b.index);
+
+const populateRoot = (array) => {
+  array.forEach((item) => {
+    const ul = document.createElement('ul');
+    const li = document.createElement('li');
+    li.innerHTML = `<div class="list">
+  <input type="checkbox" name="check" class="check">
+  <span class="description">${item.description}</span>
+  <span class="options"><i class="fas fa-ellipsis-v"></i></span>
```
Comment (henatan99) [inline src/index.js]: - [ ] Kindly align this element to the right like in the original wireframe :pray: . Everything else looks great. :+1: :clap:  

image

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @brytebee , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by barackm — APPROVED** [review]
Hi @brytebee, 👋 

# STATUS: Approved  🎉 
Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by brytebee** [comment]
Thanks for the review, I'm learning a lot 😄

**Pull request merged**